libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Settings
+page_title: Settings
+description: This article shows the seeting taht allow you to control the dataTable import/export operations. 
```

Comment:
This article shows the settings that you can use to control the DataTable import/export operations.

Reply:
fixed